tests/testthat/test-run.R

```diff
@@ -21,18 +21,13 @@ test_that("Runs work", {
   r2 <- model_summary$r.squared
   f <- model_summary$fstatistic[["value"]]
 
-  log_metric(
-    key = "R2",
-    value = r2
+  log_metrics(
+    "R2" = R2,
```

Comment:
I'd try making this test do this:

```suggestion
    R2,
```

so we can test the naming automagic

Reply:
haha literally had this and went back on it after having issues with `get_metric_history` (which are now resolved)